app/models/stats.rb

```diff
@@ -12,7 +12,10 @@ def initialize
   private
 
   def calculate_new_reg
-    25
+    start_date = (Date.today - 7).to_s
+    end_date = (Date.today - 1).to_s
+    WasteExemptionsEngine::Registration.where("submitted_at BETWEEN ? AND ?
```

Comment:
One of the things you could have done here is add a [scope](https://guides.rubyonrails.org/active_record_querying.html#scopes) to `WasteExemptionsEngine::Registration` to filter out the renewed registrations

```ruby
scope :new_registrations, -> { where("referring_registration_id IS NULL") }

# ...

WasteExemptionsEngine::Registration.new_registrations.where("submitted_at BETWEEN ? AND ?", start_date, end_date).count
```

However don't make any changes. That's more a heads up for what we might be feeding back in future PR's 😉